### ISSUE TITLE
Pre-training eval_results

### ISSUE THREAD

**Issue by ynurmahomed** [kexinhuang12345/clinicalBERT#14]
Hi,

 I am trying to replicate the results from the paper.

Would it be possible to share the `eval_results.txt` for comparison?

**Comment by kexinhuang12345** [comment]
Hey, I didn't save one. Would it be possible for you to share it over here? I can see if it makes sense

**Comment by ynurmahomed** [comment]
With 140M tokens of MIMIC-III notes, after 140K steps 128 max_seq_len + 10K steps 512 max_seq_len I get:

```
global_step = 10000
loss = 0.6517167
masked_lm_accuracy = 0.85526186
masked_lm_loss = 0.6200603
next_sentence_accuracy = 0.99
next_sentence_loss = 0.033673443
```

Loss was above 1 at 100K steps.

**Comment by kexinhuang12345** [comment]
this seems very reasonable, the accuracy is excellent and on par with what we report in the paper.
